core/plugins/org.fusesource.ide.help/.project

```diff
@@ -1,39 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
```

Comment:
any reasons for having removed the .project and .classpath files? They are useless at runtime but useful during development of Fuse Tooling. is it possible to keep them please?